storybook/stories/API/props/Tooltip.ts

```diff
@@ -30,3 +31,7 @@ export const ResponsiveProps: StorybookArgs = {
   activeDot,
   tooltipType,
 };
+
+export const ResponsivePropsRadialBar: StorybookArgs = {
```

Comment:
Instead of exporting only the tooltipType in an object, you can just export it directly, and use it as such.

Reply:
Yes, this makes good sense